packages/vue-urql/src/useQuery.ts

```diff
@@ -255,16 +253,11 @@ export function callUseQuery<T = any, V extends AnyVariables = AnyVariables>(
   const operation: Ref<Operation<T, V> | undefined> = ref();
   const extensions: Ref<Record<string, any> | undefined> = ref();
 
-  const isPaused: Ref<boolean> = isRef(_args.pause)
-    ? _args.pause
-    : ref(!!_args.pause);
+  const isPaused = ref(!!unref(args.pause));
```

Comment:
Reactivity for pause is lost due to this change.

Reply:
@kitten can you look at this? Since this is a breaking change. Since we now cannot work with pause from the hook itself, without calling pause() and resume() or modifying isPaused.